apps/pipeline/config/integration.exs

```diff
@@ -3,6 +3,7 @@ use Mix.Config
 config :pipeline,
   elsa_brokers: [{:localhost, 9092}],
   output_topic: "output-topic",
+  output_topic_prefix: "transformed",
```

Comment:
Should we be setting this property in pipeline's config?  It seems like it's more app-specific.  Also, for forklift, wouldn't this be the input topic?